NuRadioReco/modules/io/eventWriter.py

```diff
@@ -174,9 +174,9 @@ def __get_detector_dict(self, event, det):
             "channels": {},
             "stations": {}
         }
-        if is_generic_detector:
-            det_dict['default_station'] = det.get_default_station_id()
-            det_dict['default_channel'] = det.get_default_channel_id()
+        #if is_generic_detector:
```

Comment:
Please remove these instead of commenting them out.

Reply:
done